homeassistant/components/tibber/sensor.py

```diff
@@ -206,7 +197,7 @@ async def async_update(self):
             not self._tibber_home.last_data_timestamp
             or (self._tibber_home.last_data_timestamp - now).total_seconds()
             < 5 * 3600 + self._spread_load_constant
-            or not self._is_available
+            or not self._attr_available
```

Comment:
Use normal property access for this.

```suggestion
            or not self.available
```